src/Frontend/Modules/Mailmotor/Domain/Subscription/SubscribeType.php

```diff
@@ -91,7 +92,24 @@ public function getInterests(): array
     {
         $interests = [];
 
+        // should we be checking interests (CampaignMonitor for example has no interests)
+        $mailMotorInterestsNoCheck = $this->modulesSettings->get('Mailmotor', 'interests_no_check', false);
```

Comment:
the no check is for when we get a noimplemented exception from a subscriber gateway, so setting it to false when it is not filled in will always trigger the code to get the interests from the gateway